htlcswitch/switch.go

```diff
@@ -364,8 +365,8 @@ func (s *Switch) SendHTLC(nextNode [33]byte, htlc *lnwire.UpdateAddHTLC,
 	// system and something wrong happened.
 	packet := &htlcPacket{
 		incomingChanID: sourceHop,
+		outgoingChanID: firstHop,
```

Comment:
would move this to be after `incomingHTLCID` for clarity, each chanid/htlcid pair corresponds to one circuit key

Reply:
Fixed.